<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMEmployeeDetailsDialog</class>
 <widget class="QDialog" name="QMEmployeeDetailsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mitarbeiterdetails</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label_10">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Funktionen des Mitarbeiters</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="tvEmployeeFunc">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QToolButton" name="tbAddFunc">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Hinzuf端gen einer neuen Funktion</string>
           </property>
           <property name="text">
            <string>Hinzuf端gen</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbRemoveFunc">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Entfernen einer selektierten Funktion</string>
           </property>
           <property name="text">
            <string>Entfernen</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="laEmployeeName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Mitarbeiter:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Gruppe:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="laEmployeeGroup">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QLabel" name="label_11">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Schulungsausnahmen</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>20</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="tvTrainException">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QToolButton" name="tbAddTrainException">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Hinzuf端gen einer neuen Schulungsausnahme</string>
           </property>
           <property name="text">
            <string>Hinzuf端gen</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/icons/list-add.svg</normaloff>:/icons/icons/list-add.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="tbRemoveTrainException">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>Entfernen einer selektierten Schulungsausnahme</string>
           </property>
           <property name="text">
            <string>Entfernen</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/icons/list-remove.svg</normaloff>:/icons/icons/list-remove.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>26</width>
             <height>26</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLabel" name="laActivated">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QMEmployeeDetailsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QMEmployeeDetailsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbAddTrainException</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeDetailsDialog</receiver>
   <slot>addTrainException()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>340</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbRemoveFunc</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeDetailsDialog</receiver>
   <slot>removeFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>165</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbRemoveTrainException</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeDetailsDialog</receiver>
   <slot>removeTrainException()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbAddFunc</sender>
   <signal>clicked()</signal>
   <receiver>QMEmployeeDetailsDialog</receiver>
   <slot>addFunc()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>278</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>removeFunc()</slot>
  <slot>addFunc()</slot>
  <slot>removeTrainException()</slot>
  <slot>addTrainException()</slot>
 </slots>
</ui>
